tests/www/views/test_views_tasks.py

```diff
@@ -594,6 +624,36 @@ def _get_appbuilder_pk_string(model_view_cls, instance) -> str:
     return model_view_cls._serialize_pk_if_composite(model_view_cls, pk_value)
 
 
+@pytest.fixture(scope="function")
+def task_instance_to_delete(session, create_task_instance_of_operator):
+    return create_task_instance_of_operator(
+        BashOperator,
+        dag_id="unit_test_task_instance_delete",
+        task_id="test_bash",
+        bash_command="pwd",
+        execution_date=datetime.datetime(2021, 1, 1),
```

Comment:
Proposed change from @uranusjr
```suggestion
        execution_date=datetime.datetime(2021, 1, 1),
        session=session,
```